src/units/animation_component.hpp

```diff
@@ -134,4 +138,8 @@ class unit_animation_component
 
 	/** handle to the halo of this unit */
 	halo::handle unit_halo_;
+	/** handle to the abilities halos of this unit */
+	std::vector<halo::handle> abil_halos_;
+	/** vector used for check what halo_abilities vector don't modified between each reading */
```

Comment:
This needs to be proper English. I'm sort of guessing here, but do you mean this?
```suggestion
	/** vector used to check that halo_abilities vector isn't modified between each read */
```